packages/web-pkg/src/components/FilesList/ResourceListItem.vue

```diff
@@ -96,6 +92,14 @@ export default defineComponent({
       required: false,
       default: ''
     },
+    /**
+     * The resource link
+     */
+    link: {
```

Comment:
Same here. Please at least explain the difference in the docs block and when to use what.